se2_control_nodelet:
  # Vehicle parameters
  vehicle_name: "xray_source"
  mass: 15.0
  wheel_base: 0.46
  track: 0.412
  wheel_radius: 0.08
  
  # Control gains
  kx_x: 1.0
  kx_y: 1.0
  kyaw: 1.0
  
  # Initial position (optional)
  init_x: 0.0
  init_y: 0.0
  init_z: -9999.0
  
  # Use external yaw
  use_external_yaw: true 